logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/RetryFailedMessageUseCase.kt

```diff
@@ -73,7 +73,7 @@ class RetryFailedMessageUseCase internal constructor(
         messageRepository.getMessageById(conversationId, messageId)
             .flatMap { message ->
                 when (message.status) {
-                    Message.Status.FAILED -> { // TODO should it also cover status FAILED_REMOTELY?
+                    Message.Status.Failed -> { // TODO should it also cover status FAILED_REMOTELY?
```

Comment:
So, should is cover FAILED_REMOTELY?

Reply:
This is not really intent of this PR, if this is something pressing I can raise another PR after this one is merged